src/main.cxx

```diff
@@ -4,8 +4,10 @@
 #include "Exception.hxx"
 #include "LOG.hxx"
 
+#if !defined(__ANDROID__) && !defined(__EMSCRIPTEN__)
 #include <signal.h>
 void SIG_handler(int signal);
+#endif
```

Comment:
Maybe is a good time to move signal handling registration to the Exception files.

Reply:
i've create #593 for it.